### PR TITLE
Fix minor linting issues

### PR DESCRIPTION
**Pull request by alexander-demichev**
This PR fixes couple minor linting issues

### CODE REVIEW

File: pkg/cloud/gcp/actuators/util/gcp_credentials.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ import (
 	"golang.org/x/oauth2/google"
 	apicorev1 "k8s.io/api/core/v1"
 	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
```
Comment (JoelSpeed) [inline pkg/cloud/gcp/actuators/util/gcp_credentials.go:28]: Potentially would have kept `client` over `controllerclient`, but I can see that causing clashes, tough decision 🤔

Comment (Danil-Grigorev) [reply pkg/cloud/gcp/actuators/util/gcp_credentials.go:28]: I would agree that having `client` over `controllerclient` is more preferable, as we primary use the first one anywhere else.

**Pull request merged**